src/components/TabQueryEditor.vue

```diff
@@ -105,13 +105,14 @@
   import _ from 'lodash'
   import 'codemirror/addon/search/searchcursor'
   import CodeMirror from 'codemirror'
+  require('codemirror/addon/comment/comment')
```

Comment:
`import 'codemirror/addon/comment/comment'`

Reply:
Sorry, its the first project with `import/require`, i tried stuff like:
```
import Comment From '...'
```
Which dont worked but, so i thought require is the right way.
Thanks for the info :+1: 

Btw. i changed that.